pandas/tests/arithmetic/test_timedelta64.py

```diff
@@ -438,17 +467,16 @@ def test_td64arr_add_sub_timestamp(self, box):
         ts = Timestamp('2012-01-01')
         # TODO: parametrize over types of datetime scalar?
 
-        tdser = Series(timedelta_range('1 day', periods=3))
-        expected = Series(pd.date_range('2012-01-02', periods=3))
+        tdser = timedelta_range('1 day', periods=3)
```

Comment:
change the name, now that this is no longer a series?

Reply:
Sure.  There are a lot of misnomers of this form in these files